src/Ryujinx.Graphics.Nvdec.Vp9/LoopFilter.cs

```diff
@@ -96,8 +93,7 @@ internal static class LoopFilter
         };
 
         // 64 bit mask to shift and set for each prediction size.
-        private static readonly ulong[] AbovePredictionMask = new ulong[]
-        {
+        private static readonly ulong[] _abovePredictionMask = {
             0x0000000000000001UL,  // BLOCK_4X4
```

Comment:
```suggestion
            0x0000000000000001UL, // BLOCK_4X4
```
Same for other comments